cohttp/src/response.ml

```diff
@@ -43,16 +39,16 @@ let headers t = t.headers
 let encoding t = t.encoding
 let version t = t.version
 let status t = t.status
-let flush t = t.flush
 
 let make ?(version = `HTTP_1_1) ?(status = `OK) ?(flush = false)
     ?(encoding = Transfer.Chunked) ?(headers = Header.init ()) () =
+  ignore flush;
```

Comment:
```suggestion
let make ?(version = `HTTP_1_1) ?(status = `OK)
    ?(encoding = Transfer.Chunked) ?(headers = Header.init ()) () =
```

Why not dropping `flush` also from here?

Reply:
I think this would be a bigger breaking change. Since this `make` function has been around for a long time in many released versions. I'm honestly happy to get rid of it as well though if you prefer.

Reply:
I think at this point why not. Removing the flush field is already breaking, we may as well do it once and for all

Reply:
If we're talking about the 6.0 major version upgrade, I think the point is precisely to allow API breakage? One has to do some cleanup at some point, and major versions are the only opportunity we have of doing so.